packages/app/lib/users.js

```diff
@@ -51,7 +51,19 @@ const getTokenPayload = (user, userBadges) => {
     badges: { 1: userBadges[0], 2: userBadges[1], 3: userBadges[2] },
     learnPageFlag: user.learnPageFlag ? user.learnPageFlag : false,
     explorePageFlag: user.explorePageFlag ? user.explorePageFlag : false,
-    engagePageFlag: user.engagePageFlag ? user.engagePageFlag : false
+    engagePageFlag: user.engagePageFlag ? user.engagePageFlag : false,
+    bio: user.bio ? user.bio : '',
+    projects: user.projects ? user.projects : '',
```

Comment:
Why string as the default here? Wouldn't this be an array?

Reply:
My bad, and some others also, ty for the heads up. Fixed